WebApp/public/scripts/app.js

```diff
@@ -111,7 +110,6 @@ async function setupVideoPlayer(elements, config) {
   registerKeyboardEvents(videoPlayer);
   registerMouseEvents(videoPlayer, elements[0]);
   
-  startGamepadDetection();
```

Comment:
Can we use Gamepad without the line?
Need to test with Gamepad.

Reply:
Yes I moved the two lines of this function directly at the end of registerGamepadEvents(). Because the startGamepadDetection function were just registering 2 new events and was always called just after registerGamepadEvents() in all the repository and also in Furioos. I tested the gamepad both directly and through Furioos, it worked in both cases.

Reply:
Thanks.